### PR TITLE
fix: Feature service to templates

### PR DESCRIPTION
**Pull request by kevjumba**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -355,6 +365,41 @@ pprint(feature_vector)
 {% endtab %}
 {% endtabs %}
 
+## Step 7: Using a featureservice to fetch online features instead.
+
+You can also use the feature service to manage multiple features and be a central source of truth for a series of feature views. The feature store can also be used to fetch either online or historical features using the same api below. More information can be found [here](https://docs.feast.dev/getting-started/concepts/feature-service).
```
Comment (achals) [inline docs/getting-started/quickstart.md:370]:
```suggestion
You can also use feature services to manage multiple features, and decouple feature view definitions and the features needed by end applications. The feature store can also be used to fetch either online or historical features using the same api below. More information can be found [here](https://docs.feast.dev/getting-started/concepts/feature-service).
```

File: docs/getting-started/quickstart.md [hunk 2]
```diff
@@ -355,6 +365,41 @@ pprint(feature_vector)
...
+```
+
+{% tabs %}
+{% tab title="Output" %}
+```bash
+{
+ 'acc_rate': [0.5732735991477966, 0.7828438878059387],
+ 'avg_daily_trips': [33, 984],
+ 'conv_rate': [0.15498852729797363, 0.6263588070869446],
+ 'driver_id': [1004, 1005]
```
Comment (achals) [inline docs/getting-started/quickstart.md:396]: Do we expect these exact values? If not, we should caveat that the numbers here in the example may be different from what users see.

File: docs/getting-started/quickstart.md (hunk 1 above)
Comment (kevjumba) [reply docs/getting-started/quickstart.md:370]: Done.

File: docs/getting-started/quickstart.md (hunk 2 above)
Comment (kevjumba) [reply docs/getting-started/quickstart.md:396]: Done.

**Pull request merged**